### PR TITLE
Add export docs to docsv2 project

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

One high-level document about the exporting engine and what databases it supports (via which connectors), and a second document about enabling a connector.

This also adds a few connector docs and makes changes to the existing docs based on feedback from @cakrit from a few weeks ago. @vlvkobal part of this feedback was that there weren't individual pages for the JSON and Graphite connectors, which I added here. They're quite basic, but let me know if we're missing anything critical.

> See netdata/netdata-learn-docusaurus#285 for additional context.
>
> This page contains some elements that will only appear on the deployed Learn site, so see the deploy preview for this page for the full picture: https://deploy-preview-285--netdata-docusaurus.netlify.app/docs/get
>
> Some links will not work, as the target docs do not exist yet.
>
> For now, these PRs will get merged into the docsv2 but not go live on Netdata Learn. Once they're all available, or at least a workable majority, we'll merge them into master and deploy the project as a whole.

##### Component Name

area/docs
area/exporting

### CODE REVIEW

File: exporting/README.md [hunk 1]
```diff
@@ -33,51 +30,29 @@ X seconds (though, it can send them per second if you need it to).
...
+    -   [**JSON** document databases](/exporting/json/README.md)
+    -   [**OpenTSDB**](/exporting/opentsdb/README.md): Use either HTTP or HTTPS interfaces. Metrics are sent to OpenTSDB
+        as `prefix.chart.dimension` with tag `host=hostname`.
+    -   [**MongoDB**](/exporting/mongodb/README.md): Metrics are sent to the database in `JSON` format.
+    -   [**Prometheus**](/exporting/prometheus/README.md): Use an existing Prometheus installation to scrape metrics
+        from node using the Netdata API.
+    -   [**Prometheus remote write**](/exporting/prometheus/remote_write/README.md). A binary snappy-compressed protocol
+        buffer encoding over HTTP. Supports many [storage
+        providers](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+    -   [**TimescaleDB**](/exporting/TIMESCALE.md): Use acommunity-built connector that takes JSON streams from a
```
Comment (aabatangle) [inline exporting/README.md:54]: "Use acommunity-built" --> Uses a community-built

**Review by aabatangle — APPROVED** [review]

File: exporting/README.md [hunk 2]
```diff
@@ -33,51 +30,29 @@ X seconds (though, it can send them per second if you need it to).
...
-
-    -   **Google Cloud Pub/Sub Service**
-
-        metrics are sent to the service in `JSON` format.
-
-    -   **MongoDB**
-
-        metrics are sent to the database in `JSON` format.
+    -   [**JSON** document databases](/exporting/json/README.md)
+    -   [**OpenTSDB**](/exporting/opentsdb/README.md): Use either HTTP or HTTPS interfaces. Metrics are sent to OpenTSDB
```
Comment (vlvkobal) [inline exporting/README.md:46]: There is also a plaintext interface - `opentsdb:my_opentsdb_instance`.

File: exporting/README.md [hunk 3]
```diff
@@ -123,53 +99,69 @@ In `/etc/netdata/exporting.conf` you should have something like this:
...
+    collection = my_collection
+
+[json:my_json_instance]
     enabled = yes
     destination = localhost:5448
-    data source = as collected
-    update every = 2
-    send charts matching = system.active_processes
 
-[opentsdb:my_instance3]
```
Comment (vlvkobal) [inline exporting/README.md:148]: It was a valid connector configuration with a plaintext protocol.

File: exporting/README.md [hunk 4]
```diff
@@ -123,53 +99,69 @@ In `/etc/netdata/exporting.conf` you should have something like this:
...
     destination = localhost:2003
-    data source = average
-    prefix = Netdata
-    hostname = my-name
-    update every = 10
-    buffer on failures = 10
-    timeout ms = 20000
-    send charts matching = *
-    send hosts matching = localhost *
-    send names instead of ids = yes
```
Comment (vlvkobal) [inline exporting/README.md:139]: Wouldn't it be better to leave all possible options at least for one instance?

**Review by vlvkobal — CHANGES_REQUESTED** [review]

File: docs/export/enable-connector.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+Agent exports metrics as they're collected, you should start seeing data in your external database after only a few
+seconds.
+
+Any further configuration is optional, based on your needs and the configuration of your Graphite-supported database.
+See [exporting engine reference](/exporting/README.md#configuration) for details.
+
+## What's next?
+
+If you want to further configure your exporting connectors, see the [exporting engine
+reference](/exporting/README.md#configuration) for details on
```
Comment (jlbriston) [inline docs/export/enable-connector.md:84]:
```suggestion
reference](/exporting/README.md#configuration) for more details
```

File: docs/export/external-databases.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+Netdata allows you to export metrics to external time-series databases with the [exporting
+engine](/exporting/README.md). This system uses a number of **connectors** to intiate connections to [more than
+thirty](#supported-databases) supported databases, including InfluxDB, Prometheus, Graphite, ElasticSearch, and much
+more. 
+
+The exporting engine resamples Netdata's thousands of per-second metrics at a user-configurable interval, and can export
+metrics to multiple time-series databases simultaneously.
+
+Based on your needs and resources you allocated to your external time-series database, you can configure the interval at
```
Comment (jlbriston) [inline docs/export/external-databases.md:17]:
```suggestion
Based on your needs and resources you allocated to your external time-series database, you can configure the interval for
```

File: docs/export/external-databases.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+Can't find your preferred external time-series database? Ask our [community](https://community.netdata.cloud/) for
+solutions, or file an [issue on
+GitHub](https://github.com/netdata/netdata/issues/new?labels=bug%2C+needs+triage&template=bug_report.md).
+
+## What's next?
+
+We recommend you read our document on [enabling a connector](/docs/export/enable-connector.md) to learn about the
+process and discover important configuration options. If you would rather skip ahead, click on any of the above links to
+connectors for their reference documentation, which outline any prerequisites to install for that connector, along with
```
Comment (jlbriston) [inline docs/export/external-databases.md:78]:
```suggestion
connectors for their referenced documentation, which outline any prerequisites to install for that connector, along with
```

File: docs/get/README.md [hunk 8]
```diff
@@ -106,6 +111,11 @@ Netdata Cloud.
...
 and [Agent-Cloud link](/aclk/README.md) reference docs.
 
+<details>
+<summary>Watch how claiming nodes works</summary>
+<iframe width="820" height="460" src="https://www.youtube.com/embed/UAzVvhMab8g" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</details>
+
 ## What's next?
 
 At this point, you set up your free Netdata Cloud account, installed the Agent, and claimed one or more nodes to your
```
Comment (jlbriston) [inline docs/get/README.md:121]:
```suggestion
At this point, you have set up your free Netdata Cloud account, installed the Agent, and claimed one or more nodes to your
```

File: exporting/README.md [hunk 9]
```diff
@@ -7,18 +7,15 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/exporting/README
...
-metrics](/docs/export/README.md), and follow the [exporting
-quickstart](/docs/export/README.md#exporting-quickstart).
-
-The exporting engine is an update for the former [backends](/backends/README.md), which is deprecated and will be
-deleted soon. It has a modular structure and supports metric exporting via multiple exporting connector instances at the
-same time. You can have different update intervals and filters configured for every exporting connector instance. 
+The exporting engine has a modular structure and supports metric exporting via multiple exporting connector instances at
+the same time. You can have different update intervals and filters configured for every exporting connector instance. 
 
 The exporting engine has its own configuration file `exporting.conf`. Configuration is almost similar to
```
Comment (jlbriston) [inline exporting/README.md:20]:
```suggestion
The exporting engine has its own configuration file `exporting.conf`. The configuration is similar to
```

**Review by jlbriston — APPROVED** [review]

File: exporting/README.md (hunk 4 above)
Comment (joelhans) [reply exporting/README.md:139]: Good point. I added them back in to the Graphite block and explained in the paragraph above that they can be used as a reference for how more options looks/operates.

**Pull request merged**